contracts/upgradeable_contracts/OwnedUpgradeability.sol

```diff
@@ -4,14 +4,9 @@ import "../IOwnedUpgradeabilityProxy.sol";
 
 
 contract OwnedUpgradeability {
-
-    function upgradeabilityAdmin() public view returns (address) {
-        return IOwnedUpgradeabilityProxy(this).proxyOwner();
-    }
-
     // Avoid using onlyProxyOwner name to prevent issues with implementation from proxy contract
     modifier onlyIfOwnerOfProxy() {
```

Comment:
`onlyIfOwnerOfProxy` looks differently comparing to changes were made and does not reflect actual meaning.
Consider to rename it to `onlyIfUpgradeabilityOwner`